### PR TITLE
[CA-227813] Fixing some messages

### PR DESCRIPTION
**Pull request by mcintyre94**
Signed-off-by: Callum McIntyre <callumiandavid.mcintyre@citrix.com>

### CODE REVIEW

File: XenModel/Messages.resx [hunk 1]
```diff
@@ -12784,7 +12784,7 @@ A {1} user cannot alter the Workload Balancing settings.</value>
     <value>One or more hosts in the pool do not have an appropriate license.</value>
   </data>
   <data name="WLB_ERROR_4021" xml:space="preserve">
-    <value>WLB could not start a [XenServer] session with supplied invalid credentials.</value>
+    <value>WLB could not connect to [XenServer] because the supplied credentials were invalid.</value>
```
Comment (kc284) [inline XenModel/Messages.resx]: The original implies "could not open a [XenServer] session to the server", so I believe this should be "could not connect to the server".

Comment (kc284) [inline XenModel/Messages.resx]: Hmm, ok, maybe, but then it should probably be "to the [XenServer] host", just "connect to [XenServer]" sounds a bit weird to me.

Comment (MihaelaStoica) [inline XenModel/Messages.resx]: We usually try not to say "host" in the UI, we prefer "server". Although in this case "XenServer server" is not ideal :). If it makes the message clearer, then I'm OK with "XenServer host"

Comment (stephen-turner) [inline XenModel/Messages.resx]: I'm equally happy with either "connect to XenServer" or "connect to the XenServer host".

**Comment by MihaelaStoica** [comment]
Looks good. Merging.

**Pull request merged**